packages/usa-button-group/src/usa-button-group.twig

```diff
@@ -1,7 +1,47 @@
-<ul class="usa-button-group {{ modifier }}">
-  {% for item in items %}
+<!-- start segmented styles -->
+{% if segmented == true %}
+  <ul class="usa-button-group usa-button-group--segmented">
     <li class="usa-button-group__item">
-        {{ item }}
+      <button class="usa-button" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Map</button>
     </li>
+    <li class="usa-button-group__item">
+      <button class="usa-button usa-button--outline" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Hybrid</button>
+    </li>
+    <li class="usa-button-group__item">
+      <button class="usa-button usa-button--outline" {% if disabled_state == "disabled" %}disabled{% endif %} {% if disabled_state == "aria-disabled" %}aria-disabled="true"{% endif %}>Satellite</button>
+    </li>
```

Comment:
We could shorten this with a loop because we have an array of items.

```twig
  <ul class="usa-button-group usa-button-group--segmented">
    {% for item in items %}
      <li class="usa-button-group__item">
        <button class="usa-button"
          {% if disabled_state == "disabled" %}
            disabled
          {% elseif disabled_state == "aria-disabled"  %}
            aria-disabled="true"
          {% endif %}
        >
          {{ item }}
        </button>
      </li>
    {% endfor %}
  </ul>
```

Reply:
Opened issue #4881 to resolve in a future enhancement